packages/format-library/src/link/inline.js

```diff
@@ -88,6 +89,8 @@ function InlineLinkUI( {
 		]
 	);
 
+	const hasLink = linkValue?.url;
```

Comment:
Why was this moved outside of the `onChangeLink` function?

Reply:
Ah that's a left over from a previous experiment. Let me revert that.